app/react/V2/Components/Layouts/DragAndDrop/Container.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import type { FC } from 'react';
+import { IDraggable } from 'app/V2/shared/types';
+import { IDnDContext } from 'app/V2/CustomHooks';
+import { DraggableItem } from './DraggableItem';
+import { DropZone } from './DropZone';
+
+interface IItemComponentProps<T> {
+  item: IDraggable<T>;
+  context: IDnDContext<T>;
+  index: number;
+}
+
+interface ContainerProps<T> {
+  context: IDnDContext<T>;
+  itemComponent?: FC<IItemComponentProps<T>>;
+  iconHandle?: boolean;
+  className?: string;
+  parent?: IDraggable<T>;
+}
+
+// eslint-disable-next-line react/function-component-definition
+function Container<T>({
```

Comment:
If there's no inpediment to write new components as arrow functions, could you refactor them?